preliz/distributions/logitnormal.py

```diff
@@ -149,6 +149,37 @@ def kurtosis(self):
         pdf = self.pdf(x_values)
         return np.trapz(((x_values - mean) / std) ** 4 * pdf, x_values) - 3
 
+    def mode(self):
```

Comment:
This should be in the optimization module. Do you have a reference for this method? It seems to work fine for the values I manually checked. Maybe you can check how the mode is numerically computed here https://github.com/bgctw/logitnorm